lib/wash/exe/chrome.js

```diff
@@ -217,12 +307,12 @@ var isExplicitSingleTab_ = function(tabIds) {
  * @param {boolean} omitTabIds
```

Comment:
Agreed, done. I also rewrote the help, and included the previously forgotten `pluck` in there.